js/screens/Login/LoginScreen.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+
+import { connect } from 'react-redux';
+
+import { firebaseAuth } from '../../config/firebaseConfig';
+
+import { AsyncStorage } from 'react-native';
+import Login from './Login';
+import PropTypes from 'prop-types';
+
+class LoginScreen extends Component {
+  constructor() {
+    super();
+    this.state = {
```

Comment:
Please use Redux store instead of the component store.

Please do not use Redux store and the component store together, use the Redux store.